tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -36,6 +34,8 @@ const block = {
 		frontend: {
 			activeFilterType: '.wc-block-active-filters__list-item-type',
 			activeFilterName: '.wc-block-active-filters__list-item-name',
+			inStockInput: "label[for='instock']",
+			'128gbInput': "label[for='128gb']",
```

Comment:
I don't see those selectors used anywhere, maybe you didn't need them anymore?

As a suggestion, maybe let's use a selector factory here, something like:

```suggestion
			filterCheckboxFor( label: string ) => `label[for="${ label }"]`
```

Reply:
Removed the not used selectors 👍